rand_distr/src/hypergeometric.rs

```diff
@@ -126,7 +134,9 @@ impl Hypergeometric {
     /// `K = population_with_feature`,
     /// `n = sample_size`.
     #[allow(clippy::many_single_char_names)] // Same names as in the reference.
-    pub fn new(total_population_size: u64, population_with_feature: u64, sample_size: u64) -> Result<Self, Error> {
+    pub fn new(
+        total_population_size: u64, population_with_feature: u64, sample_size: u64,
+    ) -> Result<Self, Error> {
```

Comment:
We don't need this non-standard compressed layout

Reply:
Should be fixed after removal of `rustfmt.toml`.